applications/nrf_desktop/doc/ble_bond.rst

```diff
@@ -40,10 +40,10 @@ Both IDs related to the given Bluetooth peer operation are propagated in ``ble_p
 
 The identity usage depends on the device type:
 
-* nRF Desktop central
-  The nRF Desktop central uses only one application local identity and only one Bluetooth local identity (the default ones).
-* nRF Desktop peripheral
-  The nRF Desktop peripheral uses multiple local identities.
+  nRF Desktop central:
+    The nRF Desktop central uses only one application local identity and only one Bluetooth local identity (the default ones).
+  nRF Desktop peripheral:
+    The nRF Desktop peripheral uses multiple local identities.
```

Comment:
```suggestion
* nRF Desktop central
    The nRF Desktop central uses only one application local identity and only one Bluetooth local identity (the default ones).
* nRF Desktop peripheral
    The nRF Desktop peripheral uses multiple local identities.
```

Reply:
These will come out as bullet points, is that correct? I can make the changes just want to confirm that is what should be done

Reply:
Either the glossary style (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/glossary.html), where the two topics come bolded (maybe without the colons).
Or the bullet points, with the description text aligned with the text in the bullet point topic. Greg's suggestion looks now like a hybrid of those two. I guess the texts under the second option do not need to be indented.

Reply:
Okay, I will go with the latter option of using bullet points as I see now that this would match the style paragraph above.
I believe you are right too about the text under the second option not needing to be indented, so I will fix that.